src/canopy-webpack-config.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-env node */
+const path = require('path');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const merge = require('webpack-merge')
+
+module.exports = function(name, overridesConfig) {
+  if (typeof name !== 'string') {
+    throw new Error('canopy-webpack-config expects a string name as the first argument')
+  }
+
+  if (typeof overridesConfig !== 'object') {
+    throw new Error('canopy-webpack-config expects an object as a second argument to override the canopy defaults. Received ' + typeof overridesConfig)
+  }
+
+  const defaultCanopyConfig = {
+    entry: `./src/${name}.js`,
+    output: {
+      filename: `${name}.js`,
+      library: name,
+      libraryTarget: 'amd',
+      path: path.resolve(process.cwd(), 'build'),
+      chunkFilename: '[name].js',
+    },
+    mode: 'production',
+    module: {
+      rules: [
+        {
+          test: /\.js?$/,
+          exclude: [path.resolve(process.cwd(), 'node_modules')],
```

Comment:
`__dirname` is how you normally do this, but this file exists inside of projects' node_modules directories. So I'm using process.cwd() instead. This is not the preferred way to do things according to the webpack documentation because then you can't execute webpack from any directory. But I can't think of another good way to do it (passing in __dirname is not ideal either), and this will work for all of the ways we use webpack at canopy that i know of.

Reply:
It's probably okay, I don't imagine most people execute the command anywhere except in the root directory of the project.

Reply:
Even though it would be very edge case, might be good to make note of it in the readme

Reply:
updated